proto/src/serializers.rs

```diff
@@ -32,7 +32,7 @@
 //! Available serializers:
 //! i64                  <-> string:               #[serde(with="serializers::from_str")]
 //! u64                  <-> string:               #[serde(with="serializers::from_str")]
-//! std::time::Duration  <-> nanoseconds as string #[serde(with="serializers::time_duration")]
+//! core::time::Duration  <-> nanoseconds as string #[serde(with="serializers::time_duration")]
```

Comment:
Looks like this shifted the formatting and the entire second column should get an extra space.